src/panels/lovelace/components/hui-entity-editor.ts

```diff
@@ -67,15 +67,31 @@ export class HuiEntityEditor extends LitElement {
             ? ""
             : this.entities!.map((entityConf, index) => {
                 return html`
-                  <div class="entity" data-entity-id=${entityConf.entity}>
+                  <div
+                    class="entity"
+                    data-entity-id=${entityConf.entity || entityConf.type}
```

Comment:
I don't know why TS is complaining that the entity property doesn't exist here? I thought that having the OR here would make it go away...

Its because entity doesn't exist on all of the `RowConfig`s 

Any suggestions?

Reply:
Does it play better with a ternary expression? `entityConf.entity ? entityConf.entity : entityConf.type`

Reply:
I tried that too. Same issue

Reply:
`"entity" in entityConf ? entityConf.entity : entityConf.type`